src/main/java/org/terasology/climateConditions/FrostbiteSystem.java

```diff
@@ -37,9 +35,8 @@
 import org.terasology.utilities.random.Random;
 
 /**
- * Adds frostbite to the player.
- * Frostbite is a periodic effect that, e.g., damages and stuns the player.
- * Is only active iff the player has a {@link HypothermiaComponent}.
+ * Adds frostbite to the player. Frostbite is a periodic effect that, e.g., damages and stuns the player. Is only active
+ * iff the player has a {@link HypothermiaComponent}.
```

Comment:
Was putting that in one line on purpose or due to intellij formatting?

Reply:
Intellij Formatting,... just did a ctrl+alt+l before committing, but forgot about these comments... will fix them